manage.py

```diff
@@ -251,6 +256,22 @@ def purge_user(email, actor):
     permanently_delete_user(email, actor=actor)
 
 
+@click.option('--token', '-t', help="token to lookup/describe")
+@app.cli.command()
+def token_details(token):
+    valid_seconds = app.config.get(
+        'TOKEN_LIFE_IN_DAYS', 30) * 24 * 3600
```

Comment:
Can you move this implicit default to `config.py` for consistency/DRY?

Reply:
@ivan-c done - please review again when you get a chance.